package in.exception;

public class TryCatch {
	public static void main(String[] args) {

		int a = 10, b = 1;
		
		System.out.println(a / b);
	}
}
